docs/getting-started.mdx

```diff
@@ -8,46 +8,43 @@ import { AssetsProvider } from '@site/src/components/assets-context';
 import DownloadCard from '@site/src/components/download-card';
 import DownloadCardStyles from '@site/src/css/download-card.module.css';
 
+## Share your network resources within minutes
+
+Easily share private network resources with public internet users, securely, without having to alter any of your local network configuration.
+You can share resource using [NetFoundry's cloud-hosted zrok](https://myzrok.io/),
+or follow our [self-hosted guide](/docs/category/self-hosting/) to manage your own instance.
+
+* Download the binary for your platform [here](https://github.com/openziti/zrok/releases/latest)
+* Sign up for a myzrok.io account [here](https://myzrok.io/) to enable your personal share token
+* Run `zrok enable <your_token>` to enable your shell environment for sharing
+* Run `zrok share public localhost:8080` to start your first share!
+
+![zrok share public](images/zrok_share_public.png)
+
+Access your share from anywhere!
+
+```
+$ curl -s https://dslno640nct4.share.zrok.io | head
+<!DOCTYPE html>
```

Comment:
The raw HTML strikes me as visual noise I'd rather not see.

Reply:
I think I'm feeling the same. It takes up too much space and detracts from a concise intro view.